tcs-service/src/test/java/com/transformuk/hee/tis/tcs/service/service/impl/ConditionsOfJoiningServiceTest.java

```diff
@@ -37,7 +38,7 @@ void setUp() {
     ConditionsOfJoiningMapper mapper = Mappers.getMapper(ConditionsOfJoiningMapper.class);
     programmeMembershipService = mock(ProgrammeMembershipService.class);
 
-    conditionsOfJoiningService = new ConditionsOfJoiningService(repository, mapper,
+    conditionsOfJoiningService = new ConditionsOfJoiningServiceImpl(repository, mapper,
```

Comment:
Will you double check other tests & then change the name of this test class (add `Impl`) for consistency?